tests/libres_tests/integration/test_parameter_sample_types.py

```diff
@@ -146,6 +149,18 @@ def test_field_param(tmpdir, config_str, expected):
         # forward init was not set correctly
         assert load_from_forward_model(ert) == 1
 
+        fs = ert.getEnkfFsManager().getFileSystem("default_0", read_only=True)
+        if expected:
+            arr = fs.load_parameters(
+                ert.ensembleConfig(), [0], [update.Parameter("MY_PARAM")]
+            )
+            assert len(arr) == 16
+        else:
+            with pytest.raises(IndexError):
```

Comment:
Could you create an issue and perhaps a comment for this? I am quite sure it is not desired behavior, not to fix it in this PR, but so that we have it documented.